src/formats/kml/geom/KmlLineString.js

```diff
@@ -1,3 +1,7 @@
+/*
```

Comment:
There are some code format issues in this file not related to this change. Can you do a "Reformat Code"?